bin/dkr-test

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# script that runs on a host system to run tests in a container.
+#
+# To enable display of the test browser when the tests are running, the script
+# shares the hosts X display port into the container.
+#
+
+set -e
+
+SCRIPT_DIR=$( dirname "$0" )
+source "$SCRIPT_DIR/dkr-config"
+
+# build the docker command to run the tests in the container
+
+TEST_CMD="${TEST_CMD:-bash bin/test} "
+
+# this works on a linux or a mac: subsititue something based on ipconfig for windows
+ip=$(ifconfig en0 | grep inet | awk '$1=="inet" {print $2}')
+
+
+docker_cmd=" \
+  docker run -it --rm \
+  --privileged \
+  --workdir=\"/home/tester\" \
+  --volume=\"$PWD/features:/home/tester/features:ro\" \
+  --volume=\"$PWD/specs:/home/tester/specs:ro\" \
+  --volume=\"$PWD/bin:/home/tester/bin:ro\" \
+  --volume=\"$PWD/lib:/home/tester/lib:ro\" \
+  --volume=/private/tmp/.X11-unix:/tmp.X11-unix \
+  -e IN_CONTAINER=1 \
+  -e DISPLAY=\"$ip:0\" \
```

Comment:
This won't work in all cases (eg. where execution host is not display host. 
If DISPLAY is required pass it through as $DISPLAY

Reply:
Missed this earlier.   Passing through $DISPLAY didn't work on a MAC when I tried it (permission problems and symlinks as I recall) - otherwise I would not have needed the fun and games to pick out the IP address.

I take your point about a remote X-server and will have another look.